libs/damap/src/lib/components/admin/admin.component.ts

```diff
@@ -48,6 +49,10 @@ export class AdminComponent implements OnInit {
 
     const storage = await firstValueFrom(dialogRef.afterClosed());
     if (storage) {
+      if (!this.isValidUrl(storage.url)) {
+        this.feedbackService.error('http.error.storage.invalidUrl');
```

Comment:
Is it possible that you forgot to add this translation to the translation files 😜

Reply:
Nice catch :)